modules/app_component/main.tf

```diff
@@ -267,6 +267,83 @@ resource "aws_cloudwatch_log_group" "CloudWatchLogGroup" {
   }
 }
 
+# ---------------------------------------------------------------------------------------------------------------------
+# Autoscaling based on CPU and MEMORY Utilisation for scaling up and down to save costs.
+# ---------------------------------------------------------------------------------------------------------------------
+# ---------------------------------------------------------------------------------------------------------------------
+# Create autoscaling target linked to ECS
+# ---------------------------------------------------------------------------------------------------------------------
+resource "aws_appautoscaling_target" "ecs_target" {
+  max_capacity       = 3
```

Comment:
I'd like to configure autoscaling in more ways. Here, I'd like to be able to define the min and max capacities.

Reply:
Done

Reply:
Done